### PR TITLE
docs(content switcher): usage updates 

### LINKED ISSUE #1207: docs: Update Content switcher to follow new component template
Update and improve the usage documentation for content switcher. Follow new component documentation template.

### PR DESCRIPTION
**Pull request by jillianhowarth**
Closes #1207 

Usage docs updates to follow the new component documentation template.

### CODE REVIEW

File: src/pages/components/content-switcher/usage.mdx [hunk 1]
```diff
@@ -1,35 +1,43 @@
...
 title: Content switcher
-description: The content switcher is used to toggle between two or more content sections within the same space on screen. It manipulates the content shown following an exclusive or “either/or” pattern.
+description: The content switcher is used to toggle between two or more content sections within the same space on screen.
 tabs: ["Usage", "Style", "Code", "Accessibility"]
 ---
 
 <PageDescription>
 
-The content switcher is used to toggle between two or more content sections within the same space on screen. It manipulates the content shown following an exclusive or “either/or” pattern.
+The content switcher is used to toggle between two or more content sections within the same space on screen.
```
Comment (janchild) [inline src/pages/components/content-switcher/usage.mdx:9]:
```suggestion
Content switchers allow users to toggle between two or more content sections within the same space on screen.
```

File: src/pages/components/content-switcher/usage.mdx [hunk 2]
```diff
@@ -1,35 +1,43 @@
 ---
 title: Content switcher
-description: The content switcher is used to toggle between two or more content sections within the same space on screen. It manipulates the content shown following an exclusive or “either/or” pattern.
+description: The content switcher is used to toggle between two or more content sections within the same space on screen.
```
Comment (janchild) [inline src/pages/components/content-switcher/usage.mdx:3]:
```suggestion
description: Content switchers allow users to toggle between two or more content sections within the same space on screen.
```

File: src/pages/components/content-switcher/usage.mdx [hunk 3]
```diff
@@ -1,35 +1,43 @@
...
 ## Overview
 
-The content switcher is used to toggle between two or more content sections within the same space on screen. Only one section can be shown at a time.
+The content switcher is used to toggle between two or more content sections within the same space on screen. Only one content section is shown at a time.
 
-#### Text
+### When to use
 
-Be concise and specific. Titles have a max of two words.
+Use a content switcher to give users alternate views of the same or similar content. Content switchers are frequently used to let users toggle between different formatting, like with a grid view and a table view. They are also often used to narrow content groups. For example, use a content switcher if you have a single category, such as “E-mail” and you want to divide it into views such as “All”, “Read”, and “Unread”.
```
Comment (janchild) [inline src/pages/components/content-switcher/usage.mdx:32]:
```suggestion
Content switchers are frequently used to let users toggle between different formatting, like with a grid view and a table view. They are also often used to narrow content groups. For example, use a content switcher if you have a single category, such as “E-mail” and you want to divide it into views such as “All”, “Read”, and “Unread”.
```

File: src/pages/components/content-switcher/usage.mdx [hunk 4]
```diff
@@ -1,35 +1,43 @@
...
+<AnchorLink>Related</AnchorLink>
+<AnchorLink>References</AnchorLink>
 <AnchorLink>Feedback</AnchorLink>
 
 </AnchorLinks>
 
 ## Overview
 
-The content switcher is used to toggle between two or more content sections within the same space on screen. Only one section can be shown at a time.
+The content switcher is used to toggle between two or more content sections within the same space on screen. Only one content section is shown at a time.
```
Comment (janchild) [inline src/pages/components/content-switcher/usage.mdx:28]:
```suggestion
Content switchers allow users to toggle between alternate views of similar or related content. Only one content section is shown at a time.
```

File: src/pages/components/content-switcher/usage.mdx [hunk 5]
```diff
@@ -65,23 +73,94 @@ Based on usage, there should be a default selection. The default selection is al
...
+</Column>
+</Row>
+
+1. **Selected content tab:** Only one content tab can be selected at a time and there should always be one selected.
+2. **Content tab:** Selectable container for each content view.
+3. **Text label:** Describes the content view.
+
+### Sizing
+
+The overall content switcher width varies based on the content and layout. Each individual content tab should be the same width. The content tab with the longest text label should have 16px spacing to the right of the label. All other tabs should be that width as well.
```
Comment (janchild) [inline src/pages/components/content-switcher/usage.mdx:94]:
```suggestion
The overall content switcher width varies based on the content and layout. Each individual content tab should be the same width. The content tab with the longest text label should have 16px spacing to the right of the label. The width of all other tabs should match the widest tab.
```

File: src/pages/components/content-switcher/usage.mdx [hunk 6]
```diff
@@ -65,23 +73,94 @@ Based on usage, there should be a default selection. The default selection is al
...
 <Column colLg={8}>
 
-![content switcher example](images/content-switcher-usage-1.png)
+![Selected and unselected content switcher states.](images/content-switcher-usage-4.png)
 
 </Column>
 </Row>
 
-#### Content switcher vs tabs
+For detailed visual information about the various states for this component, see the Style tab.
```
Comment (janchild) [inline src/pages/components/content-switcher/usage.mdx:140]: Should we include a link here?

Comment (jillianhowarth) [reply src/pages/components/content-switcher/usage.mdx]: Good idea. Added!

**Pull request merged**